js/index.js

```diff
@@ -0,0 +1,147 @@
+    "product-h4":"Product",
+    "product-content": "Product content elementum magna eros, ac posuere elvit tempus et. Suspendisse vel tempus odio, in interdutm nisi. Suspendisse eu ornare nisl. Nullam convallis augue justo, at imperdiet metus scelerisque quis.",
+    "vision-h4":"Vision",
+    "vision-content": "Vision content elementum magna eros, ac posuere elvit tempus et. Suspendisse vel tempus odio, in interdutm nisi. Suspendisse eu ornare nisl. Nullam convallis augue justo, at imperdiet metus scelerisque quis.",
+  },
+  "contact": {
+    "contact-h4" : "Contact",
+    "address" : "123 Way 456 Street Somewhere, USA",
+    "phone" : "1 (888) 888-8888",
+    "email" : "sales@greatidea.io",
+  },
+  "footer": {
+    "copyright" : "Copyright Great Idea! 2018"
+  },
+};
+
+let randColor = () => '#'+(Math.random()*0xFFFFFF<<0).toString(16);
+
+// ~~~~~ Header ~~~~~
+const a1 = document.createElement("a");
+document.querySelector('nav').appendChild(a1);
+const a2 = document.createElement("a");
+document.querySelector('nav').prepend(a2);
+
+let headerA = document.querySelectorAll('nav a');
+headerA[0].innerHTML = "click";
+headerA[0].addEventListener('click', () => { // 1st Event Listener
+  headerA[0].style = `color: ${randColor()}`;
+});
+headerA[1].innerHTML = "mouseover";
+headerA[1].addEventListener('mouseover', () => { // 2nd Event Listener
+  headerA[1].style = `color: ${randColor()}`;
+});
+headerA[2].innerHTML = "mouseup";
+headerA[2].addEventListener('mouseup', () => { // 3rd Event Listener
+  headerA[2].style = `color: ${randColor()}`;
+});
+headerA[3].innerHTML = "dblclick";
+headerA[3].addEventListener('dblclick', () => { // 4th Event Listener
+  headerA[3].style = `color: ${randColor()}`;
+});
+headerA[4].innerHTML = 'mousemove';
+headerA[4].addEventListener('mousemove', () => { // 5th Event Listener
+  headerA[4].style = `color: ${randColor()}`;
+});
+headerA[5].innerHTML = 'auxclick';
+headerA[5].addEventListener('auxclick', () => { // 6th Event Listener
+  headerA[5].style = `color: ${randColor()}`;
+});
+headerA[6].innerHTML = 'wheel';
+headerA[6].addEventListener('wheel', () => { // 7th Event Listener
+  headerA[6].style = `color: ${randColor()}`;
+});
+headerA[7].innerHTML = 'keypress';
+document.addEventListener('keypress', () => { // 8th Event Listener
+  headerA[7].style = `color: ${randColor()}`;
+});
+
+headerA.forEach(a => a.style = 'color: green');
+
+let logo = document.getElementById("logo-img");
+logo.setAttribute('src', siteContent["nav"]["img-src"])
+
+
+// ~~~~~ CTA ~~~~~
+let ctaH1 = document.querySelector('.cta-text h1');
+ctaH1.innerHTML = siteContent['cta']['h1'];
+
+let ctaButton = document.querySelector('.cta-text button');
+ctaButton.innerHTML = siteContent['cta']['button'];
+window.addEventListener('scroll', () => {
+  ctaButton.style = `background-color: ${randColor()}`; // 9th Event Listener
+});
+ctaButton.addEventListener('mouseenter', () => {
```

Comment:
Nice work... I've gotta remember this trick for next April Fool's Day :laughing: